shapeshift/web
# feat: upsert all NFTs in nftApi

## Description

vs. missing only currently. Blockbook data is unreliable, and turns out the data we get from Alchemy/Zapper/Covalent is a lot more currently.

Accidentally, makes proper data appearing (vs. some weird empty slots) a lot faster in wallet.

## Pull Request Type


## Issue (if applicable)

N/A

## Risk



- If Alchemy or Zapper is borked, we don't have nft assets/portfolio data anymore

## Testing



- Go to Wallet
- Ensure NFTs look good and no weird empty spots with 0 balance / NFTs without a name are present
- Accidentally, this brings images for all NFTs in Wallet
- Ensure clicking NFT assets is routing to a working account asset page (test with the same URL as one of the empty slots in develop)

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

- Develop

<img width="1192" alt="Screenshot 2023-06-14 at 22 59 04" src="https://github.com/shapeshift/web/assets/17035424/869d00bd-809e-4b31-bb0c-99c4cd0900d5">
<img width="905" alt="Screenshot 2023-06-14 at 23 12 34" src="https://github.com/shapeshift/web/assets/17035424/5c44f9d0-a373-412c-9049-8affba9a992d">
- This diff

<img width="1171" alt="Screenshot 2023-06-14 at 22 58 18" src="https://github.com/shapeshift/web/assets/17035424/16b50883-9d24-4cf9-8f34-f2aff3a3dd0a">
<img width="933" alt="Screenshot 2023-06-14 at 23 12 00" src="https://github.com/shapeshift/web/assets/17035424/d1a81606-37ea-412a-8bb5-7ddaf5b9a512">
